cms/migrations/0054_add_external_coursweare_associations.py

```diff
@@ -0,0 +1,135 @@
+# Generated by Django 3.2.17 on 2023-02-24 13:09
+
+import pytz
+from datetime import datetime
+from django.db import migrations, models
+import django.db.models.deletion
+from django.contrib.contenttypes.models import ContentType
+from cms.models import ExternalProgramPage, ExternalCoursePage
+from courses.models import Course, CourseRun, Program, ProgramRun
+from ecommerce.models import Product, ProductVersion
```

Comment:
I've moved all the other models into using `apps.get_model` except for `ExternalProgramPage` in https://github.com/mitodl/mitxpro/pull/2585/commits/15a7d14f1aaf1b6a96ed16cfcd8dea0ec7444adf.

 I didn't specifically move `ExternalProgramPage` since there are some properties like `course_lineup` that we are using from that and the model acquired by `apps.get_model` this property isn't available. 

One option was to replicate all that functionality from ` course_lineup` to a function within the migration but that uses some function e.g. `get_children` from wagtail's `Page` which would also need to be replicated to the migration and that felt like making the migration too complex.

Another option was to use a custom Manager for `ExternalCoursePage` with `use_in_migrations=True` and i tried but that didn't work due to wagtail's `PageManager` so let the idea pass here.

let me know what you think.